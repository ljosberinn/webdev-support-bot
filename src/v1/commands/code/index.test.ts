import handleCodeRequest from '.';

test('sandbox replies', async () => {
  const send = jest.fn();
  const msg: any = {
    channel: { send },
  };

  await handleCodeRequest(msg);
  expect(send.mock.calls[0][0]).toMatchSnapshot();
});
