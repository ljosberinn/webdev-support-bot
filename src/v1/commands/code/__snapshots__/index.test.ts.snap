// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sandbox replies 1`] = `
"
It would be much easier to help you if we could see (parts) of your code!
Try reproducing your issue on one of these sites, save and then link it here:

<https://codepen.io/> - :warning: requires account
https://codesandbox.io/
<https://repl.it/>
<https://ideone.com/>
<https://jsfiddle.net/>

Sometimes trying to recreate a problem outside of your project already helps you tracking down the issue on your own.
"
`;
